Jenkinsfile

```diff
@@ -6,15 +6,15 @@
 
 /*
  * Copyright 2021 Joyent, Inc.
- * Copyright 2022 MNX Cloud, Inc.
+ * Copyright 2024 MNX Cloud, Inc.
  */
 
 @Library('jenkins-joylib@v1.0.8') _
 
 pipeline {
 
     agent {
-        label 'platform:true && image_ver:21.4.0 && pkgsrc_arch:x86_64 && ' +
+        label 'platform:true && image_ver:23.4.0 && pkgsrc_arch:x86_64 && ' +
```

Comment:
I do not see updated version with 24.4.0 ..

Reply:
Reload... this is  marked as outdated by github.

Reply:
yes, my bad... again..